### PR TITLE
feat: mobile / content changes

### PR DESCRIPTION
**Pull request by tlebon**
- [x] adding accordions for mobile
- [x] content updates
- [x] styling/spacing fixes
- [x] probably some other stuff

### CODE REVIEW

**Comment by tlebon** [comment]
since no one got a chance to look at this yet I'm gonna make this a draft and throw some other mobile updates/ content changes I am noticing on the new designs into it.

**Comment by tlebon** [comment]
> This pull request **introduces 2 alerts** when merging [84e755d](https://github.com/tzConnectBerlin/prediction-market-ui/commit/84e755d1c95e8024c421047f3d96ab8f831ea95c) into [8c599d5](https://github.com/tzConnectBerlin/prediction-market-ui/commit/8c599d5cd9804d2c65f552aabaf7148543e3a7d3) - [view on LGTM.com](https://lgtm.com/projects/g/tzConnectBerlin/prediction-market-ui/rev/pr-2a7f0eac1bc6c3c612e8e33ac17f214d3c033189)
> 
> **new alerts:**
> 
> * 2 for Unused variable, import, function or class

@shmodarresi  i probably introduced these but maybe you can remove them since you touched it last 🙂

**Comment by tlebon** [comment]
your changes look good to me! ill approve, but probably should let shubhendu check my changes

File: src/pages/AuctionPage/AuctionPage.tsx [hunk 1]
```diff
@@ -410,22 +413,42 @@ export const AuctionPageComponent: React.FC<AuctionPageProps> = ({ market }) =>
...
+            </Grid>
+          )}
+        </Grid>
+        {isTablet && (
+          <Grid item sm={12} xs={12} order={2} marginTop="1.5rem">
             <TradeHistory
-              columns={columnList}
+              columns={columnList.map((column) => ({
+                ...column,
+                headerAlign: 'left',
```
Comment (tlebon) [inline src/pages/AuctionPage/AuctionPage.tsx]: im not sure if its coming from here but we should center the values in these rows i think
Screen Shot 2021-10-01 at 13 34 18

Comment (shmodarresi) [reply src/pages/AuctionPage/AuctionPage.tsx]: in the design they are aligned left in all screens

**Pull request merged**